code-compass.el

```diff
@@ -1513,115 +1549,161 @@ If a file `repos-cluster.txt' exists with a list of repositories in the current
        nil
        "revisions.csv")
       (--> repository
-           c/generate-merger-script
-           c/generate-d3-v3-lib
-           c/produce-json
-           c/generate-host-enclosure-diagram-html
-           (c/run-server-and-navigate it port))))))
-
-(defun c/show-hotspot-cluster (directory date &optional port)
+           code-compass--generate-merger-script
+           code-compass--generate-d3-v3-lib
+           code-compass--produce-json
+           code-compass--generate-host-enclosure-diagram-html
+           (code-compass--run-server-and-navigate it port))))))
+(defalias 'c/show-hotspot-cluster-sync 'code-compass-show-hotspot-cluster-sync)
+
+(defun code-compass-show-hotspot-cluster (directory date &optional port)
   "Show DIRECTORY enclosure diagram for hotspots. Starting DATE reduces scope of Git log and PORT defines where the html is served."
   (interactive
    (list
     (read-directory-name "Choose repositories directory:" ".")
-    (call-interactively 'c/request-date)))
-  (c/async-run 'c/show-hotspot-cluster-sync directory date port))
+    (call-interactively 'code-compass-request-date)))
+  (code-compass--async-run 'code-compass-show-hotspot-cluster-sync directory date port))
+(defalias 'c/show-hotspot-cluster 'code-compass-show-hotspot-cluster)
 
 ;; END Hotspots for microservices
 
 ;; BEGIN display contributors
 
 
-(defun c/contributors-list-for-current-buffer ()
+(defun code-compass--contributors-list-for-current-buffer ()
   "Return contributors of this file if it is in a git repository."
   (when (vc-root-dir)
     (shell-command-to-string
      (concat
       "git shortlog HEAD -n -s -- "
       (buffer-file-name)))))
 
-(defun c/display-contributors ()
+(defun code-compass-display-contributors ()
   "Show in minibuffer the main contributors of this file."
   (interactive)
-  (when (and c/slack-main-contributor (buffer-file-name))
-    (message "Contributors of %s:\n%s" (buffer-file-name) (c/contributors-list-for-current-buffer))))
+  (when (and code-compass-slack-main-contributor (buffer-file-name))
+    (message "Contributors of %s:\n%s" (buffer-file-name) (code-compass--contributors-list-for-current-buffer))))
+(defalias 'c/display-contributors 'code-compass-display-contributors)
 
-(defun c/display-contributors-delayed ()
```

Comment:
could we leave this as public API and alias it also (`code-compass-display-contributors-delayed`)?

Reply:
done